src/autocomplete/dataset.js

```diff
@@ -38,7 +38,9 @@ function Dataset(o) {
   this.templates = getTemplates(o.templates, this.displayFn);
 
   this.css = _.mixin({}, css, o.appendTo ? css.appendTo : {});
-  this.cssClasses = _.mixin({}, css.defaultClasses, o.cssClasses || {});
+  this.cssClasses = o.cssClasses = _.mixin({}, css.defaultClasses, o.cssClasses || {});
```

Comment:
We need it here so that the assignment line 43 doesn’t fail if o.cssClasses is undefined (if you don’t pass any cssClasses option when creating the dataset, which is the case during unit tests. It should not happen in any other case as o.cssClasses is set that way in the 2 other components).
I could simply change it to
`(o.cssClasses && o.cssClasses.formattedPrefix) || _.formatPrefix(this.cssClasses.prefix, this.cssClasses.noPrefix);
` if you prefer. I did it that way as I saw it was done in other places :)